addon/mixins/sortable-item.js

```diff
@@ -320,17 +321,18 @@ export default Mixin.create({
     if (this.get('isBusy')) { return; }
 
     let drag = this._makeDragHandler(event);
+    let dragThrottled = ev => throttle(this, drag, ev, 50, false);
```

Comment:
If we want 60fps, we want this to be `1000/60 = 16`. `33` if we want 30fps. Having twiddled with settings like this before, I'm not sure what the best number is. For really complex interactions where each frame takes a long time to draw (50+ms), I've found lowering the frame rate keeps animations smoother. So, ideally, I'd love if we could find an easy way to customize this. If not, I vote for `16` instead of `50` so we're targeting 60fps